CSS/styles.css

```diff
@@ -0,0 +1,215 @@
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    -webkit-font-smoothing: antialiased; /* do I need this? */
+    background: #FFF;
+    color:#222;
+}
+
+ul {
+    list-style: none;
+}
+
+/* 
+h1, h2, h3, h3, h4 {
+
+}
+*/
+
+a {
+    color: #222;
+    text-decoration: none;
+}
+
+
+p {
+    margin: 0.5rem 0;
+}
+
+img {
+    width: 100%;
+}
+
+.content-wrap {
+    width: 90%;
+    margin: 0 auto;
+}
+
+/* Buttons */
+.btn {
+    display: inline-block;
+    cursor: pointer;
+    border: 1px solid #DDD;
+    border-radius: 4px;
+    background-color: #FFF;
+}
+
+.btn:hover {
+
+}
+
+.btn-rounded {
+	border-radius: 4px;
+}
+
+/* --------------------------- Header --------------------------- */
+header {
+    width: 100%;
+    height: 178px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 100;
+    background-color: #FFF;
+}
+
+.header-top {
+    height: 80px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.header-logo img {
+    width: 107px;
+}
+
+.header-buttons {
+    display: flex;
+    height: 48px;
+    text-align: center;
+    border-radius: 30px;
+    font-size: 15px;
+}
+
+.left-header-button {
+    padding: 15px 18px 18px;
+    font-weight: 500;
+}
+
+.middle-header-button {
+    padding: 15px 18px 18px;
+}
+
+.right-header-button {
+    padding: 15px 18px 18px;
+}
+
+#category-filter-bar {
+    z-index: 101;
+    display: flex;
+    justify-content: space-between;
+}
+
+#filters {
+    width: 90px;
+    height: 48px;
+    border-radius: 13px;
+}
+
+#next {
+    width: 28px;
+    height: 28px;
+    border-radius: 14px;
+    text-align: center;
+    line-height: 26px;
+}
+
+.categories {
+    width: 80%;
+    height: 78px;
+    overflow: hidden;
+    display: flex;
+    justify-content: space-between;
+}
+
+.category {
+    font-size: 0.8em;
+}
+
+/* --------------------------- Listings --------------------------- */
+.rentals { /* grid container */
+    position: absolute;
+    top: 200px;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: 392px 392px;
+    grid-column-gap: 25px;
+    grid-row-gap: 25px;
+}
+
+.listing { /* grid items */
+    
+}
+
+.listing-photos img { /* this will need to be responsive */
+    width: 290px;
+    height: 280px; /* the images are kinda stretched */
+    border-radius: 13px;
+}
+
+.listing-details { /* grid container */
+    display: grid;
+    grid-template-columns: 81% 16%;
+    grid-template-rows: repeat(4, 1fr);
+    grid-column-gap: 3%;
+}
+.location {
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 19px;
+}
+
+.distance, .dates {
+    color: #888888;
+    margin: 0;
+}
+
+.distance, .dates, .price {
+    grid-column-start: 1;
+}
+.price {
+    margin-top: 6px;
+}
+.rating {
+    grid-column-start: 2;
+    grid-row-start: -5;
+}
+
+/* ------------------------- Footer Links ------------------------- */
+footer {
+    background: #F9F9F9;
+    position: fixed;
```

Comment:
This probably doesn't need to be fixed position if the main content stuff is display: block and in the dom flow.